interoperator/pkg/controller/sfdefaultscheduler/sfdefaultscheduler_controller.go

```diff
@@ -86,7 +86,6 @@ type ReconcileSFDefaultScheduler struct {
 // and what is in the SFDefaultScheduler.Spec
 // TODO(user): Modify this Reconcile function to implement your Controller logic.  The scaffolding writes
 // a Deployment as an example
-// +kubebuilder:rbac:groups=osb.servicefabrik.io,resources=sfserviceinstances,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Why is it removed?

Reply:
One resource needs to be there only in atleast one controller. Otherwise the gererated yaml will have same rbac role repeating.